longevity_alternator_ttl_test.py

```diff
@@ -0,0 +1,173 @@
+import time
+
+from longevity_test import LongevityTest
+from sdcm.cluster import BaseNode
+from sdcm.utils.alternator.consts import NO_LWT_TABLE_NAME
+
+
+class AlternatorTtlLongevityTest(LongevityTest):
+
+    keyspace = f"alternator_{NO_LWT_TABLE_NAME}"
+    full_table_name = f'{keyspace}.{NO_LWT_TABLE_NAME}'
+
+    def _count_sstables_and_partitions(self, node: BaseNode) -> int:
+        estimated_num_of_partitions = node.get_cfstats(self.full_table_name)['Number of partitions (estimate)']
+        num_of_sstables = node.get_cfstats(self.full_table_name)['SSTable count']
+        self.log.info('Table stats results are: %s sstables, %s estimated partitions',
+                      num_of_sstables, estimated_num_of_partitions)
+
+        self.log.info('Run a table scan to count number of existing items')
+        with self.db_cluster.cql_connection_patient(node=node, connect_timeout=600) as session:
+            result = session.execute(f"SELECT count(*) FROM {self.full_table_name} using timeout 10m")
```

Comment:
how is it counting `sstables` or `partitions`?
it is counting the overall rows in that table, no?

Reply:
The sstables are counted and reported previously, using cfstats.
As for "partitions", this term, in Alternator, is equivalent to "items". It is just a key-value store.
So count(*) == number of partitions == number of DynamoDB items.

Reply:
ok